lib/webpack-cli.js

```diff
@@ -162,11 +162,17 @@ class WebpackCLI extends GroupHelper {
 
     runHelp(args) {
         const HelpGroup = require('./groups/help');
-        const { names } = require('./utils/commands');
-        const command = names.filter(name => {
-            return args.includes(name);
-        })[0];
-        return new HelpGroup().outputHelp(command);
+        const commandNames = require('./utils/commands').names;
+        const flagNames = require('./utils/core-flags').names;
+        const subject =
+            commandNames.filter(name => {
+                return args.includes(name);
+            })[0] ||
+            flagNames.filter(name => {
+                return args.includes(name);
+            })[0];
```

Comment:
Maybe there would be a potential scope for refactoring.

Reply:
Also, why not make use of the inline approach?

Reply:
@jamesgeorge007 refactored in latest commit.

Reply:
Much better :+1: